packages/dashboard/src/types.js

```diff
@@ -51,7 +49,7 @@ export const TemplatePropType = PropTypes.shape({
   id: PropTypes.number.isRequired,
   status: DashboardStatusesPropType,
   title: PropTypes.string.isRequired,
-  pages: PropTypes.arrayOf(StoryPropTypes.page),
+  pages: PropTypes.array,
```

Comment:
Did this type change?

Reply:
`pages` is just used to get the length of a template now.  I wanted to limit the imports of `@web-stories-wp/story-editor` so i opted to just swap it to an array since the content of that array is ditched in the serializer as soon as data's retrieved.

Reply:
🤔 Probably not worth the trouble but maybe we should update that to `numPages`?

Reply:
well we still need the pages so that we can generate stories from templates at this point.  I can add back in the dependency if this is confusing. it feels like maybe it's confusing given this thread. it just seemed redundant since it's not getting used in any component it's just getting passed.